TF-Security/Charter.md

```diff
@@ -0,0 +1,80 @@
+
+# Web of Things Security Task Force Charter
+The mission of this task force at the highest level is to ensure everything that the W3C WoT WG produces is secure by default.  Incidents such as [Mirai](https://en.wikipedia.org/wiki/Mirai_(malware)) reinforce the need to ensure security up front.
+
+## Task Force Details
+
+**End Date:** Coincident with WoT WG
+
+**Confidentiality:** Proceedings are [public](https://www.w3.org/2005/10/Process-20051014/comm.html#confidentiality-levels).
+
+**Initial Chair:** Ryan Ware
+
+**Initial Team Contact:** Dave Raggett
+
+**Meeting Schedule:** TBD
+
+
+## Table of Contents
+1. [Scope](#scope)
+2. [Deliverables](#deliverables)
+3. [Relationships to External Groups](#relationships-to-external-groups)
+4. [Participation](#participation)
+5. [Communication](#communication)
+6. [About This Charter](#about-this-charter)
+
+# Scope
+The WoT Security Task Force is responsible for:
+
+* Review of WoT related specifications for specific security relevant properties.  These include areas related to:
+    * Authentication
+    * Authorization
+    * Encryption
+    * Auditing
+* Review of use cases including:
+    * Onboarding
+    * Provisioning
+    * Compromise Recovery
+* Security Test Plans for Specifications
+* Suggested Test Plans for Implementations
+* Security Compliance Test Suite
+
+# Deliverables
+The WoT Security Task Force will not produce recommendation-track deliverables but will produce Working Group Notes on security in conformance with the [scope](#scope) of this document.  This will include specification reviews, use case reviews and security test plans.
+
+In addition to these documents, the Security Task Force will establish and continually evolve a compliance test suite.
+
+# Relationships to External Groups
+This task force will need to work with a number of different groups both within and without W3C to be successful.  Within W3C, this task force will need to work with:
+
+* [W3C Security Activity](https://www.w3.org/Security/)
+* Web of Things WG
+* [Web of Things IG](https://www.w3.org/WoT/IG/)
+* [Web of Things CG](https://www.w3.org/community/wot/)
+* [Automotive WG](https://www.w3.org/auto/wg/)
+* [Web Security IG](https://www.w3.org/2011/07/security-ig-charter.html)
+* [Web Application Security WG](https://www.w3.org/2011/webappsec/)
+
+Additionally, this task force will work with security experts from any external organization that is defined in the WoT WG charter.
```

Comment:
Should be "this task force will work with security experts from any external organization for which a liaison has been established by the WG"   And maybe it should be "WG or IG".